File: app/src/main/java/com/example/zhengshang/namepic/MyApplication.java
package com.example.zhengshang.namepic;

import android.app.Application;

import com.umeng.analytics.MobclickAgent;

/**
 * Created by zhengshang on 2017/3/10.
 */

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        MobclickAgent.setScenarioType(this, MobclickAgent.EScenarioType.E_UM_NORMAL);
        MobclickAgent.setCatchUncaughtExceptions(true);
    }
}

File: app/src/main/java/cn/zhengshang/namepic/MyApplication.kt
package cn.zhengshang.namepic

import android.app.Application

/**
 * Created by zhengshang on 2017/3/10.
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        //        MobclickAgent.setScenarioType(this, MobclickAgent.EScenarioType.E_UM_NORMAL);
//        MobclickAgent.setCatchUncaughtExceptions(true);
//        MobclickAgent.setDebugMode(true);
    }
}